bugzilla/base.py

```diff
@@ -537,14 +537,13 @@ def connect(self, url=None):
         self._transport = _RequestsTransport(
             url, self._cookiejar, sslverify=self._sslverify, cert=self.cert)
         self._transport.user_agent = self.user_agent
-        self._proxy = _BugzillaServerProxy(url, self.tokenfile,
-            self._transport)
+        self._proxy = _BugzillaServerProxy(url, self.tokenfile, self._transport)
```

Comment:
This hunk causes a ./setup.py pylint warning about > 80 columns to show up. So please drop this

Reply:
Ah, well, sorry I forgot this since "80 chars" requirement was long time gone in my world.